tstest/integration/vms/vms_test.go

```diff
@@ -197,12 +213,14 @@ func mkSeed(t *testing.T, d Distro, sshKey, hostURL, tdir string, port int) {
 			Hostname   string
 			Port       int
 			InstallPre string
+			Password   string
```

Comment:
template can be parse once and reused. As written, we parse the `user-data` and `meta-data` templates for each distro again.

you could create global vars like

```
var (
  metaDataTemplate = template.Must(template.New("meta-data.yaml").Parse(metaDataTemplate))
  userDataTemplate = template.Must(template.New("user-data.yaml").Parse(userDataTemplate))
)
```
and then use it as
```
  metaDataTemplate.Execute(...)
  userDataTemplate.Execute(...)
```